src/components/ScreenVersionSpecific.vue

```diff
@@ -39,7 +39,9 @@
             </p>
             <q-input
                 bg-color="white"
+                hint="YYYY-MM-DD"
                 label="date-released"
+                mask="####-##-##"
```

Comment:
I dont like the mask implementation, when I type `jan` my keys do not show up and I do not get any feedback that I am doing it wrong. In the old situation at least I got a regexp error message.

What I like is when you enter numbers then dashes get automatically added.

Reply:
spawned an issue #427